client/src/index.js

```diff
@@ -4,5 +4,13 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 
+// IMPORT FROM REACT-ROUTER DOM
+import { BrowserRouter, BrowserRouter as Router } from 'react-router-dom'
```

Comment:
I'm not sure why I did { BrowserRouter, BrowserRouter as Router }, it should be fine without the first "param"